dev/src/transaction.ts

```diff
@@ -333,10 +338,12 @@ export class Transaction {
    * @private
    */
   commit(): Promise<void> {
-    return this._writeBatch.commit_({
-      transactionId: this._transactionId,
-      requestTag: this._requestTag,
-    });
+    return this._writeBatch
+        .commit_({
+          transactionId: this._transactionId,
+          requestTag: this._requestTag,
+        })
+        .then(() => {});
```

Comment:
```suggestion
        .then();
```

Reply:
This doesn't compile for me and seems to forward the existing return value.

Reply:
Oh, huh. It works with `.next()`, but maybe we have a different type setup for that vs `.then()`.